### PR TITLE
nimble/ll: Minor scheduler optimizations

### PR DESCRIPTION
**Pull request by andrzej-kaczmarek**

### CODE REVIEW

**Review by wes3 — APPROVED** [review]
LGTM

**Comment by andrzej-kaczmarek** [comment]
fixes build error + rebased

File: nimble/controller/src/ble_ll_sched.c [hunk 1]
```diff
@@ -786,6 +786,22 @@ ble_ll_sched_slave_new(struct ble_ll_conn_sm *connsm)
...
 }
 
+#if MYNEWT_VAL(BLE_LL_CFG_FEAT_LL_EXT_ADV)
+static inline uint32_t
+usecs_to_ticks_fast(uint32_t usecs)
+{
+    uint32_t ticks;
+
+    if (usecs <= 31249) {
+        ticks = (usecs * 137439) / 4194304;
```
Comment (sjanc) [inline nimble/controller/src/ble_ll_sched.c:796]: Not even "John Carmack's inverse square" style comment ?:)

**Review by sjanc — APPROVED** [review]

File: nimble/controller/src/ble_ll_sched.c (hunk 1 above)
Comment (andrzej-kaczmarek) [reply nimble/controller/src/ble_ll_sched.c:796]: that's too obvious ;-)

**Pull request merged**